### PR TITLE
fix for #4049

### PR DESCRIPTION
**Pull request by mghiglieri**
Fix for #4049 and probably for #4032

### CODE REVIEW

**Comment by viggo-devries** [comment]
@mghiglieri you have lint errors, probably the removed newline.

**Comment by mghiglieri** [comment]
@viggo-devries I added a newline at the end.

File: src/oscar/apps/catalogue/expressions.py [hunk 1]
```diff
@@ -32,3 +32,4 @@ class ExpandDownwardsCategoryQueryset(Subquery):
 
     def as_sqlite(self, compiler, connection):
         return super().as_sql(compiler, connection, self.template[1:-1])
+
```
Comment (viggo-devries) [inline src/oscar/apps/catalogue/expressions.py:35]: Still a change here, please remove this change.

**Review by viggo-devries — CHANGES_REQUESTED** [review]

File: src/oscar/apps/catalogue/expressions.py (hunk 1 above)
Comment (mghiglieri) [reply src/oscar/apps/catalogue/expressions.py]: Sorry, just changed it. No changes after line 19.

**Review by viggo-devries — APPROVED** [review]

**Comment by viggo-devries** [comment]
@mghiglieri unfortunately this breaks gives a syntax error in postgres.

**Comment by specialunderwear** [comment]
you would have to make this work in both postgres and mysql @mghiglieri  this was reverted.

**Pull request merged**